src/molecules/form-fields/input-form-field.tsx

```diff
@@ -10,6 +10,8 @@ import { InputProperties } from "../../atoms/interfaces/input-properties";
 // -----------------------------------------------------------------------------------------
 
 const COMPONENT_CLASS = "c-form-field";
+export const InvalidInputFormValueClass = "-invalid";
+export const ShowLabelForScreenReadersOnlyClass = "sr-only";
```

Comment:
Actually, now that I'm seeing this, I think this is a pretty common classname we use for screenreader labels/accessibility. Do a quick search in the codebase, but we could probably promote this classname to a constant outside of this component and share it around